app/room_request/views.py

```diff
@@ -103,44 +103,49 @@ def delete_room_request(room_request_id):
     return redirect('/room-request/')
 
 
-@room_request.route('/<int:form_id>', methods=['GET', 'POST'])
-def viewID(form_id):
-    form = ActivityForm()
-    trasnferf = TransferForm()
-    user_found = False
-    comments = Activity.query.filter_by(room_request_id = form_id)
-    name = ""
-    try:
-        request_ob = RoomRequest.query.get(form_id)
-        print("User found")
-        name = str(request_ob.first_name) + " " + str(request_ob.last_name)
-        user_found = True
-    except: 
-        user_found = False
-    if form.validate_on_submit():
-            activity = Activity(
-                text=form.body.data,
-                commenter = current_user.first_name,
-                user_id = current_user.id,
-                room_request_id = request_ob.id
-                )
+@login_required
+@room_request.route('/<int:id>', methods=['GET', 'POST'])
+def view(id):    
```

Comment:
`view` is a bit more conventional naming-wise than `viewID` (in Python, we almost always use snek case :snake:)